data/json/monsters/nether.json

```
@@ -875,5 +875,59 @@
     "harvest": "exempt",
     "death_function": { "corpse_type": "NO_CORPSE", "message": "The %s melts away." },
     "flags": [ "HEARS", "GOODHEARING", "STUMBLES", "NOHEAD", "HARDTOSHOOT", "FLIES", "PLASTIC", "NO_BREATHE", "NOGIB" ]
+  },
+  {
+    "id": "mon_wraith",
+    "type": "MONSTER",
+    "name": { "str": "Wraith" },
+    "description": "A small jellyfish like creature floating in midair. You can see through it, almost as if it isn't quite fully here.",
+    "default_faction": "nether",
+    "species": [ "NETHER" ],
+    "volume": "1000 ml",
+    "weight": "0 g",
+    "hp": 10,
+    "speed": 110,
+    "symbol": "X",
+    "color": "white",
+    "aggression": 100,
+    "morale": 100,
+    "dodge": 8,
+    "special_attacks": [
+      {
+        "type": "spell",
+        "spell_data": { "id": "wraith_attack", "min_level": 1 },
+        "cooldown": 1,
+        "monster_message": "For a moment the wraith passes into you as if it were an illusion. You feel slightly more tired."
```

```suggestion
        "monster_message": "For a moment, the wraith passes into you as if it were an illusion. You feel slightly more tired."
```
Perhaps "A slight wave of fatigue washes over you"?